cg/services/run_devices/pacbio/run_file_manager/run_file_manager.py

```diff
@@ -0,0 +1,70 @@
+from pathlib import Path
+
+from cg.constants import FileExtensions
+from cg.constants.pacbio import PacBioDirsAndFiles
+from cg.services.run_devices.abstract_classes import RunFileManager
+from cg.services.run_devices.error_handler import handle_post_processing_errors
+from cg.services.run_devices.exc import PostProcessingRunFileManagerError
+from cg.services.run_devices.pacbio.run_data_generator.run_data import PacBioRunData
+from cg.services.run_devices.validators import validate_files_or_directories_exist
+from cg.utils.files import get_files_matching_pattern
+
+
+class PacBioRunFileManager(RunFileManager):
+
+    @handle_post_processing_errors(
+        to_except=(FileNotFoundError,), to_raise=PostProcessingRunFileManagerError
+    )
+    def get_files_to_parse(self, run_data: PacBioRunData) -> list[Path]:
+        """Get the file paths required by the PacBioMetricsParser."""
+        run_path: Path = run_data.full_path
+        validate_files_or_directories_exist([run_path])
+        files_to_parse: list[Path] = self._get_report_files(run_path)
+        files_to_parse.append(self._get_ccs_report_file(run_path))
```

Comment:
Why use this pattern of getting a list of stuff and then adding a last one? It's done both here and in the method below.

Reply:
The `_get_report_files` look for files inside the `unzipped_reports`, directory, the ccs_report is outside this directory. Same for the `.bam` file, it lives in another directory.